leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -191,22 +192,27 @@ var ElementInspector = Backbone.View.extend({
         this.userConstraintsHTML = $("#new-user-constraints").last().clone();
 
         // Load initial value and node name
-        this.$('.cell-attrs-text').val(this.intention.nodeName);
-        this.$('#init-sat-value').val(satisfactionValuesDict[this.intention.getInitialSatValue()].name);
+        this.$('.cell-attrs-text').val(this.model.get('intention').get('nodeName'));
```

Comment:
If you use this.model.get('intention') a lot - I would recommend in initialize putting a line "this.intention = this.model.get('intention')" so that you can use this.intention to access the intention throughout the code